code_schemes/yes_no.json

```diff
@@ -0,0 +1,92 @@
+{
+    "Version": "0.0.0.1",
+    "Name": "Yes_No",
+    "SchemeID": "Scheme-049y67c27cdcb",
+    "Codes": [{
+        "DisplayText": "Yes",
+        "VisibleInCoda": true,
+        "NumericValue": 0,
+        "CodeID": "code-ehg2ad7ce2cf",
+        "CodeType": "Normal",
+        "MatchValues": [
+          "yes"
+        ],
+        "StringValue": "yes"
+      },
+      {
+        "NumericValue": 1,
+        "CodeID": "code-41f5jk8a71ef",
+        "CodeType": "Normal",
+        "MatchValues": [
+          "no"
+        ],
+        "StringValue": "no",
+        "DisplayText": "No",
+        "VisibleInCoda": true
+      },
+      {
+        "DisplayText": "NA (missing)",
+        "VisibleInCoda": false,
+        "NumericValue": -10,
+        "CodeID": "code-NA-f78d3eb7",
```

Comment:
This is different to the id for the NA code in earlier schemes.

We kept these consistent across all schemes in ReDSS (which I think was consistent was WUSC and/or Wellcome), not sure if we still want to do that going forward though, cc @lukechurch